sdcflows/interfaces/epi.py

```diff
@@ -64,3 +66,76 @@ def _run_interface(self, runtime):
             .replace("k", "z")
         )
         return runtime
+
+
+class _SortPEBlipsInputSpec(BaseInterfaceInputSpec):
+    in_data = InputMultiObject(
+        File(exists=True),
+        mandatory=True,
+        desc="list of input data",
+    )
+    pe_dirs_fsl = InputMultiObject(
+        traits.Enum("x", "x-", "y", "y-", "z", "z-"),
+        mandatory=True,
+        desc="list of PE directions, in FSL's conventions",
+    )
+    readout_times = InputMultiObject(
+        traits.Float,
+        mandatory=True,
+        desc="list of total readout times"
+    )
+
+
+class _SortPEBlipsOutputSpec(TraitedSpec):
+    out_data = OutputMultiObject(
+        File(exists=True),
```

Comment:
https://github.com/nipy/nipype/issues/3493
```suggestion
        File(),
```